example/src/example/settings.py

```diff
@@ -9,6 +9,7 @@
 class config(SaturnConfig):
     class services_manager(ServicesManagerConfig):
         services = default_config.services_manager.services + [
+            "saturn_engine.worker.services.labels_propagator.LabelsPropagator",
```

Comment:
I would suggest to keep it in the default services? At the very least, keep it as the first service since right now the label are being propagated after the logger and tracer services.